### ISSUE TITLE
A11y: Overlay content hides other content but cannot be dismissed without moving focus

### ISSUE THREAD

**Issue by daniellekirkwood** [vector-im/element-web#24960]
<img width="605" alt="Screenshot 2023-03-27 at 09 14 42" src="https://user-images.githubusercontent.com/89144281/227881852-09fe0b40-a75b-4cac-8c3f-0e508bc2efb8.png">

**Comment by americanrefugee** [comment]
@daniellekirkwood I just spoke to Germain...

- We're using [https://www.radix-ui.com/primitives/docs/components/tooltip](https://www.radix-ui.com/primitives/docs/components/tooltip) under the hood. It lists all the properties you can use in there
- The implementation we use says "Closes when the trigger is activated or when pressing escape."
- So you could have focus, press escape, and the tooltip would disappear without your focus changing

**Comment by daniellekirkwood** [comment]
Cool, does that mean it will be handled and therefore this can be closed/actioned?

**Comment by benparsons** [comment]
I've been asked about the status of this issue, will this be handled by Radix? When will users be able to see the benefit?

**Comment by americanrefugee** [comment]
Can we change the tooltip position and style?

[Like this](https://www.figma.com/file/qd916OM7z6eYi6ea4fT0Su/WEB-IA-Concepts?type=design&node-id=1538-2538&mode=design&t=DCs0sUX3gjtuBAWc-4):

![Solution](https://github.com/vector-im/element-web/assets/112560169/cfb3529f-4b21-41bc-9938-cfdfd4a54dd6)

**Comment by t3chguy** [comment]
@americanrefugee we can by switching to Compound/Radix tooltips

**Comment by americanrefugee** [comment]
@daniellekirkwood Does this resolve the issue? Not sure if I should close it or assign this to somebody now...

**Comment by daniellekirkwood** [comment]
It would appear that the work needed to close this issue is to switch to compound tooltips -- @t3chguy I've seen other issues on this. Do you know if we've made the switch or not yet?

**Comment by daniellekirkwood** [comment]
Either way, that's the fix so removing the designers' assignment

**Comment by t3chguy** [comment]
@daniellekirkwood its a case of replacing it on a per-tooltip basis, I think 2 or 3 sites now use Compound tooltips but the vast vast majority do not yet.

**Comment by daniellekirkwood** [comment]
Ok, thanks. So this issue stays open until we replace all tool tips. 

@Johennes this is an A11y issue for customers, we might need to discuss pulling this into Q4 somehow.

**Comment by Johennes** [comment]
If this falls under retainer, that shouldn't be an issue?

**Comment by t3chguy** [comment]
Worth noting we really ought not to adopt Compound tooltips until https://github.com/vector-im/compound/issues/248 is remedied

**Comment by emmaburton1** [comment]
> Yes, I just need to confirm that this is for a customer. @emmaburton1 or @benparsons do you know if there's a customer retainer ask for this? **Be aware that this issue is in our external facing repo** so feel free to take this to a Room if needed.

Yes - please count this as a retainer request. (Further discussion in the Element Web Lobby.)
